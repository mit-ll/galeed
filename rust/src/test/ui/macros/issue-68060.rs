fn main() {
    (0..)
        .map(
            #[target_feature(enable = "")]
            //~^ ERROR: attribute should be applied to a function
            //~| ERROR: the feature named `` is not valid for this target
            //~| NOTE: `` is not valid for this target
            #[track_caller]
            //~^ ERROR: `#[track_caller]` requires Rust ABI [E0737]
            |_| (),
            //~^ NOTE: not a function
        )
        .next();
}
