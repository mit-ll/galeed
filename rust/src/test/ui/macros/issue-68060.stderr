error: attribute should be applied to a function
  --> $DIR/issue-68060.rs:4:13
   |
LL |             #[target_feature(enable = "")]
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
...
LL |             |_| (),
   |             ------ not a function

error: the feature named `` is not valid for this target
  --> $DIR/issue-68060.rs:4:30
   |
LL |             #[target_feature(enable = "")]
   |                              ^^^^^^^^^^^ `` is not valid for this target

error[E0737]: `#[track_caller]` requires Rust ABI
  --> $DIR/issue-68060.rs:8:13
   |
LL |             #[track_caller]
   |             ^^^^^^^^^^^^^^^

error: aborting due to 3 previous errors

For more information about this error, try `rustc --explain E0737`.
